webapp/src/modules/rental/selector.spec.ts

```diff
@@ -134,3 +136,41 @@ describe('when getting if the transaction is being submitted', () => {
     })
   })
 })
+
+describe("when getting rental status", () => {
```

Comment:
The word `a` is missing here, as we need to use an article:
```suggestion
describe("when getting a rental confirmation status", () => {
```
Also, I've added the `confirmation` word so it doesn't get confused with the rental status.